### ISSUE TITLE
Error: prop attribute not handled

### ISSUE THREAD

**Issue by nilskuhn** [tjk/vue-o2c#11]
I am getting error
```
> vue-o2c@1.0.0 start /home/<my-project-path>/vue-o2c
> node index.js

/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:44
  throw new Error(`${msg}${n ? ` @ (${n.startPosition.row + 1}, ${n.startPosition.column + 1})` : ""}`);
  ^

Error: prop attribute not handled: required @ (6, 17)
    at fail (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:44:9)
    at Object.onKeyValue (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:614:19)
    at handleObject (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:1058:53)
    at Object.onKeyValue (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:604:11)
    at handleObject (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:1058:53)
    at handleProps (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:593:3)
    at handleDefaultExportKeyValue (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:954:7)
    at Object.onKeyValue (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:1100:30)
    at handleObject (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:1058:53)
    at handleDefaultExport (/home/<my-project-path>/vue-o2c/node_modules/.pnpm/vue-o2c@0.1.21_web-tree-sitter@0.20.8/node_modules/vue-o2c/dist/index.js:1099:3)
 ELIFECYCLE  Command failed with exit code 1.
```
trying to transform this vue3 option API component:
```
<template>
  <div class="offer-count">
    <h3 v-if="!loading">{{ count }}{{ countText }}</h3>
  </div>
</template>

<script>
export default {
  name: 'DetailsOffersCount',
  props: {
    count: {
      type: Number,
      required: true
    },
    loading: {
      type: Boolean,
      required: true
    }
  },
  computed: {
    countText() {
      return ` ${this.count === 1 ? 'Angebot' : 'Angebote'} sortiert nach: `
    }
  }
}
</script>
```
Looks quite similar, if I use `npx vue-o2c /path/to/component.vue`:
```
Error: prop attribute not handled: required @ (6, 17)
    at fail (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:37:9)
    at Object.onKeyValue (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:607:19)
    at handleObject (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:1051:53)
    at Object.onKeyValue (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:597:11)
    at handleObject (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:1051:53)
    at handleProps (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:586:3)
    at handleDefaultExportKeyValue (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:947:7)
    at Object.onKeyValue (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:1093:30)
    at handleObject (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:1051:53)
    at handleDefaultExport (/home/nkuhn/.npm/_npx/9a0efcde2cb26dd4/node_modules/vue-o2c/dist/cli.js:1092:3)
```

**Comment by tjk** [comment]
Thanks for reporting! This should be a simple fix... let me take a quick look.

**Comment by tjk** [comment]
Should be fixed in [0.1.22](https://github.com/tjk/vue-o2c/releases/tag/0.1.22). Thanks for reporting, please let me know if you run into other issues!
